x/execution/internal/types/params.go

```diff
@@ -1,38 +1,66 @@
 package types
 
 import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/params"
+	"github.com/cosmos/cosmos-sdk/x/params/subspace"
 )
 
 // Default parameter namespace
 const (
 	DefaultParamspace = ModuleName
+	DefaultMinPrice   = "10000atto"
 )
 
-// ParamKeyTable for instance module
-func ParamKeyTable() params.KeyTable {
-	return params.NewKeyTable().RegisterParamSet(&Params{})
-}
+var (
+	// KeyMinPrice key for the parameter MinPrice
+	KeyMinPrice = []byte("MinPrice")
+)
 
 // Params - used for initializing default parameter for instance at genesis
-type Params struct{}
+type Params struct {
+	MinPrice string `json:"min_price" yaml:"min_price"` // min price to pay for an execution
```

Comment:
```suggestion
	MinPrice string `json:"minPrice" yaml:"minPrice"` // min price to pay for an execution
```
why snake case here?